### ISSUE TITLE
Icons for our Job Board

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/opensourcedesign.github.io#156]
As recommended in #140 it'd be great if we could get some icons for specific jobs that people can choose from. 

Suggested categories: 
* Mobile App
* Website
* Hardware
* Desktop App

We _could_ use something like font awesome in the mean time, but if we want to release our own icon set this might be a neat place to start.

**Comment by AmanDeep9925** [comment]
@simonv3 and @RDIL i want to work on this issue can you assign me this?

**Comment by RDIL** [comment]
@AmanDeep9925 sure it’s all yours

**Comment by itzsoumyadip** [comment]
What to do , with this issue ?

**Comment by RDIL** [comment]
@itzsoumyadip as described by @simonv3, you would need to add icons for different job categories like Mobile Apps, websites, etc.

**Comment by itzsoumyadip** [comment]
Ok .. can I add icon from (font awesome) ..?

**Comment by RDIL** [comment]
That should work.

**Comment by jancborchardt** [comment]
Yup, we already use Font Awesome via the main.css so that would be great. :) Thanks @itzsoumyadip!

**Comment by A-GHOSH-dev** [comment]
Can I work for this issue here?

**Comment by jdittrich** [comment]
You can, but I do not exactly know what is needed here, maybe the linked  #140 explains.
